src/Impostor.Tools.ServerReplay/Program.cs

```diff
@@ -114,6 +120,17 @@ private static ServiceProvider BuildServices()
 
         private static async Task ParseSession(BinaryReader reader)
         {
+            var protocolVersion = reader.ReadInt32();
+            if (protocolVersion != 0)
```

Comment:
`protocolVersion < 1`

Reply:
`< 1` or `> 0`

Reply:
`> 0`

Reply:
Actually no because you will need to invert the if statement then lol